src/masternodes/rpc_accounts.cpp

```diff
@@ -1203,6 +1203,91 @@ UniValue listaccounthistory(const JSONRPCRequest& request) {
     return slice;
 }
 
+UniValue getaccounthistory(const JSONRPCRequest& request) {
+    auto pwallet = GetWallet(request);
+
+    RPCHelpMan{"getaccounthistory",
+               "\nReturns information about account history.\n",
+               {
+                    {"owner", RPCArg::Type::STR, RPCArg::Optional::NO,
+                        "Single account ID (CScript or address)."},
+                    {"blockHeight", RPCArg::Type::STR, RPCArg::Optional::NO,
+                        "Height to iterate from (downto genesis block)."},
+                    {"txn", RPCArg::Type::STR, RPCArg::Optional::NO,
+                        "for order in block."},
+               },
+               RPCResult{
+                       "{}  An object with account history information\n"
+               },
+               RPCExamples{
+                       HelpExampleCli("getaccounthistory", "cscript 160 10")
+                       + HelpExampleCli("getaccounthistory", "cscript, 160, 10")
+               },
+    }.Check(request);
+
+    std::string accountId = request.params[0].getValStr();
+    CScript owner = DecodeScript(accountId);
+    uint32_t blockHeight = std::stoul(request.params[1].getValStr());
+    uint32_t txn = std::stoul(request.params[2].getValStr());
```

Comment:
use `.get_int()` since it has runtime errors defined.

Reply:
What if we change input param [1] and [2] type to RPCArg::Type::NUM, then it would be more efficient, right ?